akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -956,13 +956,15 @@ private[stream] final class GroupedWithin[T](n: Int, d: FiniteDuration) extends
   override def initialAttributes = DefaultAttributes.groupedWithin
   val shape = FlowShape(in, out)
 
-  override def createLogic(inheritedAttributes: Attributes): GraphStageLogic = new TimerGraphStageLogic(shape) {
+  override def createLogic(inheritedAttributes: Attributes): GraphStageLogic = new TimerGraphStageLogic(shape) with InHandler with OutHandler {
```

Comment:
yeah, although I introduced this to save on memory (this is one of the reason for the ThreadLocal interpreter, so these methods can provide default implementations for the finish/fail methods and still be stackable traits)